configs/kgs-bot/vit-bot.cfg

```diff
@@ -0,0 +1,63 @@
+# Our bots only trained with Tromp-Taylor rules. This setting makes GTP ignore
+# commands that change the rules.
+ignoreGTPRuleChanges = true
+
+# 65k visits with 64 threads should be slightly weaker than 32k visits with 1 thread.
+maxVisits = 65536
+numSearchThreads = 64
```

Comment:
try merging https://github.com/AlignmentResearch/KataGo-custom/pull/111, I think it makes throughput stronger.

 * 1 GPU can get 65536 visits + 64 threads in ~20 seconds
* 2 GPU can get 131072 visits + 96 threads in ~20 seconds

Reply:
Done.